pepper-apis/src/test/java/org/broadinstitute/ddp/db/FormInstanceDaoTest.java

```diff
@@ -41,13 +41,10 @@
     private static String studyGuid;
 
     private static ActivityInstanceService activityInstanceService;
-    private static FormInstanceDao dao;
 
     @BeforeClass
     public static void setup() {
-        SectionBlockDao sectionBlockDao = new SectionBlockDao();
-        dao = FormInstanceDao.fromDaoAndConfig(sectionBlockDao, sqlConfig);
-        org.broadinstitute.ddp.db.ActivityInstanceDao activityInstanceDao = new org.broadinstitute.ddp.db.ActivityInstanceDao(dao);
+        org.broadinstitute.ddp.db.ActivityInstanceDao activityInstanceDao = new org.broadinstitute.ddp.db.ActivityInstanceDao();
```

Comment:
Should we move these tests into the test file for `ActivityInstanceService` and then get rid of this `FormInstanceDaoTest` file?

Reply:
Done